classification/classification_network.py

```diff
@@ -75,10 +73,17 @@
     "pamLength": 3,
 }
 
+
+# TODO: replace with SEQ_CLASSIFIER_PARAMS
 CLA_CLASSIFIER_PARAMS = {"steps": "1",
                          "implementation": "py"}
+KNN_CLASSIFIER_PARAMS = {
```

Comment:
I added this in b/c I like using it to run validation experiments; the kNN classifier is very intuitive as to what the exact classification probabilities should be, and the `compute` method works.